pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -133,6 +133,8 @@ func (e *cloudWatchExecutor) newSession(region string) (*session.Session, error)
 		cfgs = append(cfgs, &aws.Config{Endpoint: aws.String(dsInfo.Endpoint)})
 	}
 
+	var err error
+	var sess *session.Session
```

Comment:
Nit, but I don't see the point in assigning to `sess` and `err` in this scope within the `switch` block, since those assignments won't be used (outside of the switch block).
```suggestion
```

Reply:
good catch!